data-model/ELASTICSEARCH/index_mappings.md

```diff
@@ -1,43 +1,44 @@
-curl -XPUT '$YOUR_INDEX_URL:9200/wherehows' -d '
-{
-  "mappings": {
-    "dataset": {},
-    "comment": {
-      "_parent": {
-        "type": "dataset"
-      }
-    },
-    "field": {
-      "_parent": {
-        "type": "dataset"
-      }
-    }
-  }
-}
-'
 
-curl -XPUT '$YOUR_INDEX_URL:9200/wherehows/flow_jobs/_mapping' -d '
+https://www.elastic.co/guide/index.html
+
```

Comment:
Please explain the intention and convention of using alias "wherehows" here.

The create step index step needs to reflect the usage of alias.